core/ibft.go

```diff
@@ -1099,19 +1119,33 @@ func (i *IBFT) validPC(
 		return false
 	}
 
+	//Make sure all have the same round
```

Comment:
Nitpick: comments should be separated from `//` with a space

Reply:
Solved